BaseTools/Conf/build_rule.template

```diff
@@ -350,6 +353,14 @@
         $(CP) ${dst} $(BIN_DIR)(+)$(MODULE_NAME_GUID).efi
         -$(CP) $(DEBUG_DIR)(+)*.map $(OUTPUT_DIR)
         -$(CP) $(DEBUG_DIR)(+)*.pdb $(OUTPUT_DIR) 
+
+    <Command.CLANGDWARF>
+        $(CP) ${src} $(DEBUG_DIR)(+)$(MODULE_NAME).debug
+        "$(GENFW)" -e $(MODULE_TYPE) -o ${dst} ${src} $(GENFW_FLAGS)

```

Comment:
Is there a reason why GENFW is not the first line, like in the preceding stanza?

Reply:
That is reasonable, just the only mentioned other profile in the commit message is CLANGPDB.